src/Microsoft.Health.Dicom.SqlServer/Features/CustomTag/CustomTagLimit.cs

```diff
@@ -0,0 +1,41 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+using System.Collections.Generic;
+using Dicom;
+
+namespace Microsoft.Health.Dicom.SqlServer.Features.CustomTag
+{
+    /// <summary>
+    /// Limits on CustomTag feature.
+    /// </summary>
+    internal static class CustomTagLimit
+    {
+        /// <summary>
+        /// Mapping from CustomTagVR to DateType
+        /// </summary>
+        public static readonly IReadOnlyDictionary<string, CustomTagDataType> CustomTagVRAndDataTypeMapping = new Dictionary<string, CustomTagDataType>()
```

Comment:
I think this should just be inside of `SqlCustomTagStore`, as there don't seem to be any other consumers of this dictionary.

Reply:
it will be used by STOW and QIDO